README.md
#projectTed

history.txt
   12  node a.js
   13  node a.js
   14  node a.js
   15  node a.js
   16  node a.js
   17  node a.js
   18  node a.js
   19  node a.js
   20  node a.js
   21  node a.js
   22  node a.js
   23  node a.js
   24  node a.js
   25  node a.js
   26  node a.js
   27  node a.js
   28  node a.js
   29  node a.js
   30  node a.js
   31  npm a.js
   32  npm start
   33  git status
   34  npm run dev
   35  npm run dev
   36  ls
   37  cd models
   38  ls
   39  touch Products.js
   40  cd ..
   41  npm run dev
   42  ls
   43  cd client/
   44  ls
   45  cd src/components/
   46  node a.js
   47  npm run dev
   48  git status
   49  git commit -a -m 'minor changes'
   50  git push origin master
   51  cd ..
   52  mkdir Redux_Practice
   53  npm create-react-app Redux_Practice/
   54  create-react-app Redux_Practice/
   55  create-react-app redux_practice/
   56  npm i redux --save-dev
   57  mkdir -p src/js/store
   58  ls
   59  ls src
   60  ls -a js/
   61  ls -a js/store
   62  ls -a js
   63  ls -a src
   64  ls -a src/js
   65  cd src/js
   66  ls
   67  cd store
   68  touch index.js
   69  mkdir -p src/js/reducers
   70  cd ..
   71  mkdir reducers
   72  cd reducers/ && touch index.js
   73  cd ..
   74  mkdir -p action && touch index.js
   75  rmdir action
   76  mkdir -p actions && cd actions && touch index.js
   77  cd ..
   78  mkdir -p constants && cd constants && touch types.js
   79  cd ..
   80  touch index.js
   81  npm start
   82  npm start
   83  npm start
   84  npm start
   85  npm start
   86  npm start
   87  cd ..
   88  cd ..
   89  npm i react-redux -s
   90  mkdir -p src/js/components
   91  ls
   92  cd src/js/components/
   93  ls
   94  ls
   95  touch List.jsx
   96  cd ../../
   97  cd ..
   98  npm start
   99  npm start
  100  npm start
  101  npm start
  102  ls src
  103  ls src
  104  npm start
  105  npm start
  106  cd src
  107  cd components && touch Form.jsx
  108  cd js/components && touch Form.jsx
  109  cd ../..
  110  cd ..
  111  npm i uuid --save-dev
  112  cd src
  113  ls
  114  cd ..
  115  npm start
  116  mkdir -p src/js/middleware
  117  cd src/js/middleware
  118  touch index.js
  119  cd ../..
  120  cd ..
  121  npm start
  122  cd ..
  123  mkdir -p newRedux
  124  create-react-app newredux
  125  mkdir store && cd store && store.js
  126  touch store.js
  127  cd ..
  128  mkdir reducers && cd reducers && touch reduce.js
  129  cd src 
  130  cd 
  131  ls
  132  cd Desktop/images/
  133  ls
  134  cd newRedux/
  135  l
  136  ls
  137  cd src
  138  mkdir reducers && cd reducers && touch reduce.js
  139  rm reduce.js 
  140  touch reducer.js
  141  cd ..
  142  mkdir actions && cd actions && touch types.js
  143  cd ..
  144  mkdir constants && cd constants && touch types.js
  145  cd ..
  146  cd ..
  147  npm i react-redux --save-dev
  148  ls
  149  mkdir components && cd components && touch App.js
  150  cd..
  151  cd 
  152  cd Desktop/images/newRedux/sr
  153  cd Desktop/images/newRedux/src
  154  mkdir components && cd components && touch App.js
  155  touch List.js
  156  touch Form.js
  157  cd ..
  158  cd 
  159  cd Desktop/images/newRedux/
  160  npm i uuid --save-dev
  161  npm start
  162  npm i react-redux save
  163  npm start
  164  npm i redux save
  165  npm start
  166  npm i redux-thunk --save-dev
  167  npm i redux-thunk --save-dev
  168  npm start
  169  npm run dev
  170  ls
  171  cd client/compo
  172  cd client/src/components/
  173  touch SearchBar.js
  174  touch Product.js
  175  cd ..
  176  cd ..
  177  cd ..
  178  git commit -a -m 's'
  179  git push origin master
  180  npm run dev
  181  npm run dev
  182  cd client/
  183  ls
  184  cd src 
  185  ls
  186  cd components/
  187  touch AddBid.js
  188  npm start
  189  npm i
  190  npm start
  191  npm start
  192  npm run dev
  193  npm i uuid --save-dev
  194  npm run dev
  195  git commit -a -m "post "
  196  git push origin master
  197  npm run dev
  198  npm run dev
  199  npm run dev
  200  npm run dev
  201  git commit -a -m "added new bid to database"
  202  git push origin master
  203  npm run dev
  204  npm run dev
  205  git commit -a -m "active bids OK"
  206  git push origin master
  207  npm run dev
  208  node xml.js
  209  node xml.js ./items-0.xml
  210  node xml.js ./items-0.xml
  211  node xml.js ./items-0.xml
  212  node xml.js ./items-0.xml
  213  node xml.js ./items-0.xml
  214  node xml.js ./items-0.xml
  215  node xml.js ./items-0.xml
  216  node xml.js ./items-0.xml
  217  node xml.js ./items-0.xml
  218  node xml.js ./items-0.xml
  219  node xml.js ./items-0.xml
  220  node xml.js ./items-0.xml
  221  node xml.js ./items-0.xml
  222  node xml.js ./items-0.xml
  223  node xml.js ./items-0.xml
  224  node xml.js ./items-0.xml
  225  node xml.js ./items-0.xml
  226  node xml.js ./items-0.xml
  227  node xml.js ./items-0.xml
  228  node xml.js ./items-0.xml
  229  node xml.js ./items-0.xml
  230  node xml.js ./items-0.xml
  231  node xml.js ./items-0.xml
  232  node xml.js ./items-0.xml
  233  node xml.js ./items-0.xml
  234  node xml.js ./items-0.xml
  235  node xml.js ./items-0.xml
  236  node xml.js ./items-0.xml
  237  node xml.js ./items-0.xml
  238  node xml.js ./items-0.xml
  239  node xml.js ./items-0.xml
  240  node xml.js ./items-0.xml
  241  node xml.js ./items-0.xml
  242  node xml.js ./items-0.xml
  243  node xml.js ./items-0.xml
  244  node xml.js ./items-0.xml
  245  node xml.js ./items-0.xml
  246  node xml.js ./items-0.xml
  247  node xml.js ./items-0.xml
  248  node xml.js ./items-0.xml
  249  node xml.js ./items-0.xml
  250  node xml.js ./items-0.xml
  251  node xml.js ./items-1.xml
  252  node xml.js ./items-2.xml
  253  node xml.js ./items-3.xml
  254  node xml.js ./items-4.xml
  255  node xml.js ./items-5.xml
  256  node xml.js ./items-6.xml
  257  node xml.js ./items-7.xml
  258  node xml.js ./items-8.xml
  259  node xml.js ./items-9.xml
  260  node xml.js ./items-10.xml

xml2.js
var XMLWriter = require('xml-writer');
xw = new XMLWriter;
var format = require('xml-formatter');

xw.startDocument();
xw.startElement('Items');
xw.startElement("Item");
xw.writeAttribute('ItemID', 5);

//Name
xw.startElement("Name");
xw.text(20);
xw.endElement();

//Currently
xw.startElement("Currently");
xw.text(20);
xw.endElement();

//First_Bid
xw.startElement("First_Bid");
xw.text(20);
xw.endElement();

//Number_of_Bids
xw.startElement("Number_of_Bids");
xw.text(21);
xw.endElement();

//Location
xw.startElement("Location");
xw.writeAttribute("Latitude", 22);
xw.writeAttribute("Longitude", 24);
xw.text(24);
xw.endElement();

//Country
xw.startElement("Country");
xw.text(25);
xw.endElement();

//Started
xw.startElement("Started");
xw.text(444);
xw.endElement();

//Ends
xw.startElement("Ends");
xw.text(555);
xw.endElement();

//Description
xw.startElement("Description");
xw.text(555);
xw.endElement();

//Seller
xw.startElement("Seller");
//xw.text(bid.);
xw.writeAttribute('Rating', 66);
xw.writeAttribute('UserID', 88);
xw.endElement();



xw.text('Some content');
xw.endElement("Item");
xw.endDocument();
var formattedXml = format(xw.toString());
console.log(formattedXml);

console.log(xw.toString());